### PR TITLE
Update learning rate accordingly with changes on TensorFlow side

### PR DESCRIPTION
**Pull request by lc0**
Updated tutorial accordingly to https://github.com/OpenMined/PySyft/issues/2346

Btw, would it also make sense to add "run in colab" link to make it easier for people to experiment

### CODE REVIEW

**Comment by lc0** [comment]
I also saved from colab directly, hence a bit updated structure, but I am open for feedback

**Comment by lc0** [comment]
or shall I push in original jupyter format? 
so in this case it would be easier to review PR - since the diff would be smaller

**Comment by jvmancuso** [comment]
Yes, please resubmit this PR after modifying the existing Jupyter notebook!

**Comment by jvmancuso** [comment]
I would also suggest that we just switch the optimizer -- there's no particular reason that Adadelta was used there, and so I'd suggest replacing with SGD + Nesterov Momentum or Adam to be closer to best practices

**Comment by lc0** [comment]
@jvmancuso what do you think about making these ipynb more approachable with a link `run in colab`?

**Comment by lc0** [comment]
@jvmancuso just changed to Adam. 

I still think it would be valuable over time add ability to run notebooks directly in colab

**Comment by lc0** [comment]
ping

**Comment by jvmancuso** [comment]
running on colab would be sweet! we'd welcome a commit here with that change (or a separate PR if you prefer)

**Comment by lc0** [comment]
@jvmancuso what is the process of merging these PRs once they are approved?

**Comment by jvmancuso** [comment]
Process is (1) update the PR branch with most recent commits from dev, (2) resolve any merge conflicts), and (3) wait for someone with write access to merge (although sometimes a gentle reminder here or in Slack can be helpful).  In this case, I was waiting to hear back to see if you wanted to add commits for the colab link to this one.  I assume that'll be a separate PR though, so will merge now.

**Comment by lc0** [comment]
@jvmancuso please go ahead and merge this one and I am going to open another PR that adds a link to run on colab for bunch of notebooks. This way we keep it more consistent
